Events/src/test/java/com/github/swcraftlyon/meetup/infra/dao/MangoEventDaoTest.java

```diff
@@ -0,0 +1,35 @@
+package com.github.swcraftlyon.meetup.infra.dao;
+
+import com.mongodb.client.FindIterable;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.Filters;
+import org.bson.Document;
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class MangoEventDaoTest {
+
+
+    @Test
+    public void shouldGetMangoDataBase() throws Exception {
+        Assert.assertNotNull(MangoEventDao.getDataBase());
+    }
+
+    @Test
+    public void shouldPersistInDataBase() throws Exception {
+        MongoDatabase mongoDatabase = MangoEventDao.getDataBase();
```

Comment:
Oupps !! Sorry ! i test that i can insert something in db but not MangoEventDao. i will so it.